hw7.py

```diff
@@ -0,0 +1,26 @@
+phonebook = {}
+while True:
+    user_input = input('Enter command: ')   # format command name phone
+    split_input = user_input.split()
+    command = split_input[0]
+    name = split_input[1]
+    phone = split_input[2]

```

Comment:
Your program breaks here if `add someone` command entered

Reply:
I dont understand what breaks, it works as following: 
![image](https://user-images.githubusercontent.com/119344933/210527105-8afb503a-eeab-4bb9-b4da-6ade5516ffd4.png)

Reply:
I agree here